awx/main/tests/unit/test_tasks.py

```diff
@@ -1656,6 +1656,7 @@ def test_multi_cloud(self):
                 'password': 'secret'
             }
         )
+        azure_rm_credential.inputs['secret'] = ''
```

Comment:
These changes are here because `decrypt_field` is no longer accessing credential inputs through `__getattr__` and therefore will not implicitly pick up the V1Credential.FIELDS default (typically '') for undefined inputs.

This should be ok provided we weren't specifically testing that behavior here. @ryanpetrello thoughts?